tests/FakeItEasy.Tests/Creation/ProxyGeneratorSelectorTests.cs

```diff
@@ -92,16 +93,15 @@ public void Should_delegate_calls_to_default_generator_asking_about_non_delegate
             var getHashCode = fake.GetType().GetMethod("GetHashCode");
 
             string reason = null;
-            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason))
-                .Returns(true).AssignsOutAndRefParameters("reason");
+            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason)).Returns(true).AssignsOutAndRefParameters("reason");

```

Comment:
I'll merge as is, but I think this line break should have stayed present. Note how the old statement is completely visible in the GitHub diff UI but the new one spills over out of sight to the right.

Indeed, I've sent a few PR's with refactoring commits which limit line lengths to ~120 chars. (Our [guidelines](https://github.com/FakeItEasy/FakeItEasy/blob/master/CONTRIBUTING.md#line-width) say ~160, but I think we should shorten that to ~120 which allows some breathing room for the diff UI, which seems to spill over at around 130.)